tests/sentry/tasks/post_process/tests.py

```diff
@@ -412,6 +416,34 @@ def test_owner_assignment_owner_is_gone(self):
         assignee = event.group.assignee_set.first()
         assert assignee is None
 
+    @patch("sentry.tasks.servicehooks.process_service_hook")
+    def test_event_parameter_backwards_compat(self, mock_process_service_hook):
```

Comment:
@markstory We can remove this test later, yes? If so, probably should have a TODO here.

Reply:
Yeah it can be removed in a month when self-hosted gets cut again.